package.json

```diff
@@ -6,9 +6,10 @@
     "lint": "cross-env NODE_ENV=development && eslint . --cache --ext .js,.jsx,.ts,.tsx",
     "lint:fix": "cross-env NODE_ENV=development eslint . --cache --ext .js,.jsx,.ts,.tsx --fix",
     "test": "jest",
-    "test-e2e": "yarn build && node -r @babel/register ./config/scripts/CheckBuildsExist.js && cp -R ./build/* ./src/electron && cross-env NODE_ENV=test testcafe electron:./src/electron ./tests-e2e && rm -rf ./src/electron/dist ./src/electron/main.js",
+    "test-e2e:no-build": "node -r @babel/register ./config/scripts/CheckBuildsExist.js && cp -R ./build/* ./src/electron && cross-env NODE_ENV=test testcafe electron:./src/electron ./tests-e2e && rm -rf ./src/electron/dist ./src/electron/main.js",
```

Comment:
do we need this new command since we're running testcafe directly in the actions? might be useful to leave it here though.